multihost_test/Bugzillas/test_error_2006_0032.py

```diff
@@ -0,0 +1,67 @@
+""" Automation of Shadow Utils tests """
+
+from __future__ import print_function
+import pytest
+
+
+class TestShadowUtilsErrors():
+    """
+    Automation of Shadow Utils tests
+    """
+    @pytest.mark.tier1
+    def test_error_2006_0032(self, multihost):
+        """
+        :title: Shadow-Utils: Tests if shadow-utils are
+        immune against bugs in 2006:0032
+        :id: 017f615a-92a8-11eb-bca1-002b677efe14
+        :steps:
+          1. Creating tmp directory
+          2. Allowing '.' in usernames
+          3. Added two patches to shadow-utils for
+          additional usermod flag
+          4. The user foo must be included in both
+          foo-group and foo-group2 groups
+          5. usermod -p does not update sp_lstchg
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+          5. Should succeed
+        """
+        # Creating tmp directory
+        cmd = multihost.client[0].run_command("mktemp -d")
+        assert cmd.returncode == 0
+        # useradd not allowing '.' in usernames
+        cmd = multihost.client[0].run_command("useradd "
+                                              "joe.testing")
+        assert cmd.returncode == 0
+        multihost.client[0].run_command("userdel "
+                                        "-r joe.testing")
+        # Added two patches to shadow-utils for
+        # additional usermod flag
+        multihost.client[0].run_command("useradd foo")
+        multihost.client[0].run_command("groupadd foo-group")
+        multihost.client[0].run_command("usermod -G "
+                                        "foo-group foo")
+        multihost.client[0].run_command("groupadd foo-group2")
+        multihost.client[0].run_command("usermod -G "
+                                        "foo-group2 -a foo")
+        # The user foo must be included in both
+        # foo-group and foo-group2 groups
+        cmd = multihost.client[0].run_command("id foo")
+        assert cmd.returncode == 0
```

Comment:
I think there's a check missing here. You should check that `foo` is included in both `foo-group` and `foo-group2`

Reply:
Fixed